docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -158,27 +158,78 @@ installation directory to the `Path` user environment variable.
+| `keymap.nextTab`              | `Command+Tab` on macOS<br/>`Ctrl+Tab` on Windows/Linux                                                                                                              | Shortcut to go to the next tab.                                        |
+| `keymap.openConnections`      | `Command+P` on macOS<br/>`Ctrl+P` on Windows/Linux                                                                                                                  | Shortcut to open the connections panel.                                |
+| `keymap.openClusters`         | `Command+E` on macOS<br/>`Ctrl+E` on Windows/Linux                                                                                                                  | Shortcut to open the clusters panel.                                   |
+| `keymap.openClusters`         | `Command+E` on macOS<br/>`Ctrl+E` on Windows/Linux                                                                                                                  | Shortcut to open the clusters panel.                                   |
+| `keymap.openProfiles`         | `Command+I` on macOS<br/>`Ctrl+I` on Windows/Linux                                                                                                                  | Shortcut to open the profiles panel.                                   |
+| `keymap.openCommandBar`       | `Command+K` on macOS<br/>`Ctrl+K` on Windows/Linux                                                                                                                  | Shortcut to open the command bar.                                      |
+| `terminal.fontFamily`         | `Menlo, Monaco, 'Courier New', monospace` on macOS<br/>`'Consolas', 'Courier New', monospace` on Windows<br/>`'Droid Sans Mono', 'Courier New', monospace` on Linux | Font family for the terminal.                                          |
+| `terminal.fontSize`           | 15                                                                                                                                                                  | Font size for the terminal.                                            |
+
+<Admonition
+  type="note"
+  title="Note"
+>
+  In the config file there is an additional `$schema` property. It is a link to the JSON schema that
+  describes the available config options.
+  It allows editors to provide autocompletion.
+</Admonition>
+
+### Configuring keyboard shortcuts
+A valid shortcut contains at least one modifier and a single key code, for example `Shift+Tab`.
+Function keys (`F21`-`F24`) do not require a modifier.
+Modifiers and a key code must be combined by `+` character.
+
+<Tabs>
+  <TabItem label="macOS">
+    Available platform modifiers: `Control`, `Option`, `Shift`, `Command`.
+  </TabItem>
+  <TabItem label="Windows/Linux">
+    Available platform modifiers: `Ctrl`, `Alt`, `Shift`.
+  </TabItem>
+</Tabs>
```

Comment:
```suggestion
Available modifiers:

* `Control`, `Option`, `Shift`, `Command` on macOS.
* `Ctrl`, `Alt`, `Shift` on Windows and Linux.
```

`<Tabs>` here IMHO obscures too much information which can be shown in a simpler way without this component.